new_core_concepts/modules/ROOT/pages/typeql/query-variables-patterns.adoc

```diff
@@ -112,45 +112,45 @@ insert
   ## $p is a `person`, who has the attribute $n
   $p isa person; $p has $n;
 
-  ## $e is an 'employment', where '$p' plays the 'employee' role
+  ## $e is an 'employment', where $p plays the 'employee' role
   $e isa employment; $e links (employee: $p);
 ----
 [NOTE]
 ====
 Variables are also bound to concepts when they are inserted by an `isa` constraint.
-See xref:{page-version}@new_core_concepts::typeql/schema-data.adoc[]
+See xref:{page-version}@new_core_concepts::typeql/schema-data.adoc[].
 ====
 
 == A quick note on variables
 Variable names are unique within a pipeline -
 every usage of a variable with a certain name refers to the same variable.
 
-In the following sections, the term [[definition-local-variable]]"local" to mean
+In the following sections, the term [[definition-local-variable]]"local" is used to mean
 a variable is bound in that pattern but not in its parent.
 This contrasts with the usage in programming languages where it refers to the scope of a variable name.
-As we have seen, the scope of variable names in TypeQL is the pipeline.
+As we have seen, the scope of variable names in TypeQL is the xref:{page-version}@new_core_concepts::typeql/query-clauses.adoc[pipeline].
```

Comment:
Maybe we don't need this sentence after all, since we only said it two sentences ago.
Else: 
```suggestion
As mentioned earlier, the scope of variable names in TypeQL is the xref:{page-version}@new_core_concepts::typeql/query-clauses.adoc[pipeline].
```